src/components/RadioGroup/RadioGroup.jsx

```diff
@@ -0,0 +1,136 @@
+import _ from 'lodash';
+import classNames from 'classnames';
+import React from 'react';
+
+import { bindClassNames } from '../../util/style-helpers';
+import { createLucidComponentDefinition } from '../../util/component-definition';
+import getRandom from '../../util/random';
+
+import RadioButton from '../RadioButton/RadioButton';
+import reducers from './RadioGroup.reducers';
+
+const boundClassNames = bindClassNames('RadioGroup');
+
+const {
+	func,
+	node,
+	number,
+	string
+} = React.PropTypes;
+
+/**
+ * {"categories": ["controls", "toggles"]}
+ *
+ * This is a group of related radio buttons whose values are mutually exclusive
+ * and one whom must be selected any given moment in time.
+
+ * Any props that are not explicitly defined in `propTypes` are spread onto the
+ * root element.
+ */
+const RadioGroup = React.createClass(createLucidComponentDefinition({
+	displayName: 'RadioGroup',
+
+	childProps: {
+		RadioButton: { ...RadioButton.propTypes }
+	},
+
+	reducers,
+
+	propTypes: {
+		/**
+		 * Should be instances of `RadioGroup.RadioButton` which supports the
+		 * same props as `RadioButton`.
+		 */
+		children: node,
```

Comment:
I think it should definitely be `node` which represents any "renderable" type (strings, numbers, elements or an array of those types). Anything else isn't renderable so wouldn't be able to be passed in as a child, anyway.